osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -46,6 +46,12 @@ public class Catcher : SkinReloadableDrawable, IKeyBindingHandler<CatchAction>
 
         public Container ExplodingFruitTarget;
 
+        public Container<DrawableHitObject> CaughtFruitContainer { get; } = new Container<DrawableHitObject>
```

Comment:
I'd have preferred these to be exposed via a `public Drawable CreateProxiedContent()` method instead, rather than exposing containers directly.

Reply:
in catcherarea too?

Reply:
That one is probably okay since it's not a container.

Reply:
so `CatcherArea.Catcher.CreateProxiedContent()`?

Reply:
Yeah, that looks fine.